scripts/basic-workflow-tests.sh

```diff
@@ -399,6 +452,12 @@ test_exec() {
 
     task Evaluating the output of ilab model train
     test_evaluate
+
+    if [ "${PHASED_TRAINING}" -eq 1 ] && [ "${TRAIN_LIBRARY}" -eq 1 ]; then
+        test_phased_train
+    elif [ "${PHASED_TRAINING}" -eq 1 ] && [ "${TRAIN_LIBRARY}" -eq 0 ]; then
+        echo "-P requires -T. Skipping phased training."
```

Comment:
This should probably be enforced, you can put it in `set_defaults` where other sanity checks for CLI arguments are done already.

Reply:
fixed